### DISCUSSION TITLE
已经下载过的I怎么咋地

### DISCUSSION THREAD

**Discussion by holore** [Dineshkarthik/telegram_media_downloader#321] [General]
因为我删除了一个不能播放的视频，希望能重新下载，但是重新运行没有反应，我想知道程序是怎么知道哪些已经下载过了
Sorry，I just a rookie

**Comment by tangyoha** [comment]
@holore  hi, in `config.yaml`, `ids_to_retry`表示失败的下载列表数组，`last_read_message_id`表示最后一次成功下载的文件，请查看你的项目中文件配置。
